browser/templates/browser/coursegroup.html

```diff
@@ -168,7 +168,8 @@ <h3>Attendance</h3>
     .remove();
 
   u.on('click', function(d) {
-      window.location.href = '/course/' + d.id + '/';
+      // Split the url to preserve the part before coursegroup and added the clicked course from the graph to url
+      window.location.href = window.location.href.split('/coursegroup')[0] + '/course/' + d.id + '/';
```

Comment:
is there a reason you're not using the "courses" url here (with the url tag)?

Reply:
Added courses url, and appended clicked course number from the graph with it